inc/tag_replacer.php

```diff
@@ -137,7 +152,11 @@ public function process_image_tags( $content, $images = array() ) {
 
 		$image_sizes = self::image_sizes();
 		$sizes2crop  = self::size_to_crop();
-
+		self::$filters[ Optml_Settings::FILTER_TYPE_LAZYLOAD ][ Optml_Settings::FILTER_CLASS ] = apply_filters(
```

Comment:
we don't need to initialize this, it's already done here https://github.com/Codeinwp/optimole-wp/blob/d21e4628128d0e5215bc3a47adb02f28b799af4e/inc/app_replacer.php#L271

The filter is not required here, we can use this one instead to exclude some image tags in various compatibilities https://github.com/Codeinwp/optimole-wp/blob/d21e4628128d0e5215bc3a47adb02f28b799af4e/inc/app_replacer.php#L127

To keep consistency, we can add the saved classes list to this filter `optml_possible_lazyload_flags` and remove the `contains_banned_lazyload_class` method.

Reply:
ok, I modified it now